extract_metadata.py

```diff
@@ -0,0 +1,111 @@
+from dandi.dandiapi import DandiAPIClient
+import pandas as pd
+from pathlib import Path
+
+modalities = {'oct': 'PS-OCT',
+              'df': 'Dark field microscopy',
+              'xpct': 'HiP-CT',
+              'dwi': 'Diffusion MRI',
+              'fluo': 'Light sheet microscopy',
+              'photo': 'Blockface photo'}
+
+# Create dataframe of all assets across all datasets
+def extract_assets():
+    client = DandiAPIClient("https://api.lincbrain.org/api")
+    client.dandi_authenticate()
+    dandisets = client.get_dandisets()
+
+    print(f"Processing {sum(1 for _ in dandisets)} Datasets on lincbrain.org")
+
+    df = pd.DataFrame(columns=["Dataset",
+                            "Version",
+                            "Subject", 
+                            "Modality",
+                            "Path",
+                            "Filename",
+                            "Extension",
+                            "Directory", # Top-level directory (e.g. source data, raw data, derivatives)
+                            'Size (bytes)'])
+
+    for dataset in dandisets:
```

Comment:
Could you exclude `'000048', '000004'` here rather than a few lines below?

Reply:
I have moved the if statement above.  Let me know if you had something else in mind.